### PR TITLE
Run.d: allow hooks for processes during container start

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: container/root/run.sh [hunk 1]
```diff
@@ -1,30 +1,11 @@
 #!/bin/bash
-CONFIG_SITE=/etc/nginx/sites-available/default
-CONFIG_SERVER=/etc/nginx/nginx.conf
+RUN_SCRIPTS=/run.d
```
Comment (mannytoledo) [inline container/root/run.sh]: :tada: Nice!

**Comment by mannytoledo** [comment]
:+1:

**Comment by iderdik** [comment]
:+1:  Great idea

Can this also be accomplished with ONBUILD ?

**Comment by bryanlatten** [comment]
@iderdik these are runtime-specifc, onbuild only happens at build-time (where env variables are not possible)

**Comment by Aintaer** [comment]
:+1:

**Pull request merged**